Tests/SentryTests/Integrations/Performance/CoreData/SentryCoreDataTrackerTest.swift

```diff
@@ -236,6 +236,22 @@ class SentryCoreDataTrackerTests: XCTestCase {
         XCTAssertEqual(transaction.children[0].status, .internalError)
     }
     
+    func test_Request_with_Error_is_nil() {
+        let fetch = NSFetchRequest<TestEntity>(entityName: "TestEntity")
+        
+        let transaction = startTransaction()
+        let sut = fixture.getSut()
+        
+        let context = fixture.context
+        
+        let _ = try?  sut.fetchManagedObjectContext(context, request: fetch, isErrorNil: true) { _, _ in
+            return nil
+        }
```

Comment:
A lot of the test cases in here use `try?`, but could be strengthened with `XCTAssertNoThrow`/`XCTAssertThrowsError`.

```suggestion
        XCTAssertNoThrow(try sut.fetchManagedObjectContext(context, request: fetch, isErrorNil: true) { _, _ in
            return nil
        })
```